### PR TITLE
selectdevice returns MULTI:device in cumulative_throughput

### PR DESCRIPTION
**Pull request by wgzintel**
### Details:
 - *Implement cumulative by calling multi*
 - *...*

### Tickets:
 - *ticket-75212*

### CODE REVIEW

File: src/plugins/auto/executable_network.cpp [hunk 1]
```diff
@@ -179,14 +179,31 @@ MultiDeviceExecutableNetwork::MultiDeviceExecutableNetwork(const std::string&
...
-            _loadContext[ACTUALDEVICE].networkPrecision, _context.modelPriority);
+
+    if (isCumulative) {
+        std::list<DeviceInformation> validDevices =
+            _multiPlugin->GetValidDevice(metaDevices, _loadContext[ACTUALDEVICE].networkPrecision);
+
+        std::string deviceName = "MULTI:";
+        for (auto& device : validDevices) {
+            deviceName += device.deviceName;
+            deviceName += ((device.deviceName == validDevices.back().deviceName) ? "" : ",");
```
Comment (tiger100256-hu) [inline src/plugins/auto/executable_network.cpp:197]: should load MULTI:CPU,GPU with the performanceHint throughput

Comment (tiger100256-hu) [reply src/plugins/auto/executable_network.cpp:197]: for CUMULATIVE_THROUGHPUT， cpu_help should be disabled

**Review by wangleis — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
how does cumulate throughput work if I use `core.compile_model(model, "GPU", ov::hint::performance(cumulative))` ? 
Will it select several GPUs at once if I have 2 discrete GPU on my machine? 
Or the logic covers cumulative only in AUTO mode?

**Comment by wangleis** [comment]
In 2022.2 release, cumulate throughput (ctput) only enabled in AUTO.
In 2022.3 release, will extend ctput to all device. Then `core.compile_model(model, "GPU", ov::hint::performance(cumulative))` will use all GPUs on the system.

**Pull request merged**